corremap.m~

```diff
@@ -0,0 +1,15 @@
+function corrmap(data,sel,unit,type)
+    % [] = RASTER_PLOT(data,sel,opt,type)
+    % data - given struct array 
```

Comment:
What does this function contain? It's just comments

Reply:
Deleted, old version